src/vision/scripts/listener.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# Software License Agreement (BSD License)
```

Comment:
Ok, I think this is just an example subscriber that we're not using, so you should be good to delete it.